exercises/phone-number/.meta/src/reference/java/PhoneNumber.java

```diff
@@ -34,6 +34,14 @@ private String normalize(String number) {
                 throw new IllegalArgumentException("Can only have 11 digits if number starts with '1'");
             }
         }
+        
+        if (number.startsWith("0") || number.startsWith("1")){
+            throw new IllegalArgumentException("Illegal character in phone number. "
+                    + "Only digits, spaces, parentheses, hyphens or dots accepted.");
+        } else if (number.charAt(3) == '0' || number.charAt(3) == '1'){
+            throw new IllegalArgumentException("Illegal character in phone number. "
+                    + "Only digits, spaces, parentheses, hyphens or dots accepted.");
+        } 
```

Comment:
I don't think this is the correct error message considering what you're checking for 🙂